querier.go

```diff
@@ -161,7 +161,7 @@ func (q *Querier) Stop() {
 
 func (q *Querier) queryLabels(ctx context.Context) {
 	queryStart := time.Now()
-	resp, err := q.client.Labels(ctx, connect.NewRequest(&queryv1alpha1.LabelsRequest{}))
+	resp, err := q.client.Labels(ctx, connect.NewRequest(queryv1alpha1.LabelsRequest{}))
```

Comment:
```suggestion
	resp, err := q.client.Labels(ctx, connect.NewRequest(&queryv1alpha1.LabelsRequest{}))
```
This is already fixed locally again.